app/gui2/src/util/navigator.ts

```diff
@@ -70,6 +70,11 @@ export function useNavigator(viewportNode: Ref<Element | undefined>) {
     return { x: -x + w / 2, y: -y + h / 2 }
   })
 
+  const translate = computed(() => ({
+    x: transformValue.value.x,
+    y: transformValue.value.y,
```

Comment:
This computed is unncessary. The `transformValue` should simply be renamed to `translate`.